### DISCUSSION TITLE
setting password policies in local dev environment

### DISCUSSION THREAD

**Discussion by khera** [supabase/supabase#30478] [Questions] [answered]
On the Supabase hosted instances, I can set password policies for my users: minimum length, complexity, plus even a captcha. There is nothing related to these settings in the CLI configuration documentation at https://supabase.com/docs/guides/local-development/cli/config#auth-config

Is there a way to set these on local dev so I can test these out before I push to staging/production?

The closest thing I found is https://github.com/orgs/supabase/discussions/7181 for setting minimum length with self- hosting in the docker configuration, but that's not possible in local dev.

**Comment by silentworks** [comment, marked as answer]
The minimal length and complexity aren't currently exposed to the CLI. ~~I can't seem to find anything related to captcha in the GoTrue readme, so not sure if this is related to auth.~~ I can get a PR opened for the minimal length and complexity to get this added to the CLI.

edit: I've found the captcha related stuff too. I will PR the captcha stuff separately.
